packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -20,6 +20,7 @@ export interface ValidatorGroup {
   name: string
   url: string
   members: Address[]
+  affiliates?: Address[]
```

Comment:
I'd say, either we add functionality to the wrapper to add the affiliates if a flag is given, or we remove this, and handle the different type in the cli commands 

Which can be done with:
```ts
const validatorGroup: Array<ValidatorGroup & { affiliates?: Address[]}> = await validators.getValidatorGroup(res.args.groupAddress)
``

Either way, this `?` on type definitions tend to be cumbersome later, when you have a value that **you know it has affiliates** but the type doesn't express that, so, you need to constantly check `group.affiliates != null` to access it.

So I'd probably just remove this here